numpy/core/src/private/ufunc_override.h

```diff
@@ -242,9 +242,12 @@ PyUFunc_CheckOverride(PyUFuncObject *ufunc, char *method,
         }
         tmp = PyArray_GetAttrString_SuppressException(obj, "__array_ufunc__");
         if (tmp) {
+            /* compare with &PyArray_Type.tp_methods[3].ml_meth??? */
```

Comment:
This is where I need the right comparison of the actual method, i.e., something better rather than just bailing out on an object being `ndarray`.